core/modules/simpletest/tests/cache.test

```diff
@@ -327,3 +327,86 @@ class CacheIsEmptyCase extends CacheTestCase {
     $this->assertTrue($cache->isEmpty(), 'The cache bin is empty');
   }
 }
+
+/**
+ * Test the cache tagging system.
+ */
+class CacheTagsUnitTestCase extends BackdropUnitTestCase {
```

Comment:
The test says it can't find the `invalidated_cache_tags` table. Maybe this is because this test uses `BackdropUnitTestCase` where it might not run the system module update hooks? Just a guess.

Reply:
Hm, I can get it to pass locally, and there doesn't seem to be anything in the test that would need the db.

Reply:
Sorry, it's not this test, but the `upgrade_bare.test` that's causing the failures. Maybe because by default `use_cache_tags_cache` is set to `TRUE` but db creation doesn't happen until much later in the upgrade.

Reply:
I added another check for the table, so this should not happen during the upgrades any more.